### ISSUE TITLE
Azure Spatial Anchors 2.10.1 for iOS is missing CFBundleVersion

### ISSUE THREAD

**Issue by RuiBEAT** [Azure/azure-spatial-anchors-samples#275]
The current package 2.10.1 for iOS seems to be invalid for distribution, according to the message provided by XCode during the distribution of the package through App Store Connect.

The project builds and runs just fine in both Android and iOS devices, but uploading the package through App Store Connect makes XCode display the following error message:

```
"This bundle Payload/Onirix.app/Frameworks/AzureSpatialAnchors.framework is invalid. The Info.plist file is missing the required key: CFBundleVersion
```

![XCodeError](https://user-images.githubusercontent.com/20771303/126495275-4c498274-4bf2-4c3c-90a6-e30a8c66376e.png)

Checking the aforementioned Info.plist and comparing it with one of version 2.9.0, the value of the indicated key is indeed empty. Left is for version 2.9.0, right for version 2.10.1.

![Captura de pantalla 2021-07-21 a las 15 23 34](https://user-images.githubusercontent.com/20771303/126496015-ff67c0f5-3eea-4ce9-a0c4-4bd69b186117.png)

Opening the Info.plist from the framework and filling manually the empty field with the same "1" value fixes the problem.

Regards,

**Comment by MehranAzimi-msft** [comment]
Thanks for reporting this issue. We are investigating and are looking into addressing it in a hot fix, and will keep this thread updated.

**Comment by MehranAzimi-msft** [comment]
We published ASA SDK 2.10.2 hotfix that contains a fix for this issue.

**Comment by oafolabi-msft** [comment]
@RuiBEAT , would you say we can close this issue, or do you still run into some problems?
